examples/elements/bokeh/Box.ipynb

```diff
@@ -41,17 +41,19 @@
    },
    "outputs": [],
    "source": [
-    "%%opts Box (line_width=5 color='purple') Image (cmap='gray')\n",
+    "%%opts Box (line_width=5 color='red') Image (cmap='gray')\n",
     "data = np.sin(np.mgrid[0:100,0:100][1]/10.0)\n",
-    "data[range(30, 70), range(30, 70)] = -3\n",
-    "hv.Image(data) * hv.Box(-0, 0, 0.5 )"
+    "data[range(40, 60) + range(40,50), range(20, 40)+ range(70,80)] = -3\n",
```

Comment:
Not sure tbh, np.arange but then you have to concatenate, ``list`` is also ugly.